File: src/main/java/io/github/dqualizer/dqlang/types_new/adapter/constants/loadprofile/LoadProfile.java
package io.github.dqualizer.dqlang.types_new.adapter.constants.loadprofile;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
public class LoadProfile {

    @JsonProperty("load_peak")
    private LoadPeak loadPeak;
    @JsonProperty("load_increase")
    private LoadIncrease loadIncrease;
    @JsonProperty("constant_load")
    private ConstantLoad constantLoad;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/k6adapter/dqlang/constants/loadprofile/LoadProfile.kt
package io.github.dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class LoadProfile {
    @JsonProperty("load_peak")
    val loadPeak: io.github.dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile.LoadPeak? = null

    @JsonProperty("load_increase")
    val loadIncrease: io.github.dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile.LoadIncrease? = null

    @JsonProperty("constant_load")
    val constantLoad: io.github.dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile.ConstantLoad? = null
}
